conf/webpack.config.js

```diff
@@ -69,9 +69,9 @@ function getConfig(isReactExternalized) {
         module: {
             rules: [
                 {
-                    test: /\.js$/,
+                    test: /\.(js|mjs)$/,
                     loader: 'babel-loader',
-                    exclude: /(node_modules)/,
+                    exclude: /node_modules\/pikaday/,
```

Comment:
pikaday was throwing warnings

Reply:
Should we instead move the rule from the `isRelease` condition below to a root rule? Does the pikaday issue not show up for production builds?

Reply:
Using the rule from below had implications in dev. I can try again, but the one below seemed for production mode

Reply:
Also we cant use the one below as-is, since for dev we want sourcemaps and all

Reply:
I wonder what else its pulling in due to IE9 and IE10
https://github.com/facebook/create-react-app/blob/master/packages/babel-preset-react-app/dependencies.js#L90

And still testing but we want this not included for non-IE11 builds
https://github.com/facebook/create-react-app/blob/master/packages/babel-preset-react-app/dependencies.js#L113